src/index.js

```diff
@@ -1,27 +1,74 @@
 import './style.css';
 
 const todoList = document.querySelector('.todo-list');
-const todos = [
-  {
-    index: 1,
-    completed: true,
-    task: 'Wash the dish',
-  },
-  {
-    index: 2,
-    completed: true,
-    task: 'Go to the gym',
-  },
-];
+const clearBtn = document.querySelector('.clear-btn');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+function saveTasks() {
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+function removeTask(id) {
+  todos = todos.filter((task) => task.id !== id);
+  saveTasks();
+}
 
 function displayTasks() {
-  todos.forEach((todos) => {
+  todoList.innerHTML = '';
+
+  todos.forEach((task) => {
     const li = document.createElement('li');
     li.className = 'list-item';
+    li.innerHTML = `
+      <input type="checkbox" class="check" />
+      ${task.task}
+      <span class="material-icons close" data-id="${task.id}">close</span>
+    `;
```

Comment:
According to the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.), there should be a functionality to edit the tasks. Consider adding this function to satisfy the milestone requirement.